### ISSUE TITLE
problem with uploading OSM

### ISSUE THREAD

**Issue by Carap1209** [TraceyLauriault/COMS2200A#32]
#problems with uploading OSM

I followed the instruction and edit my pitures to OSM. On the last step when I press the upload button, it showed errors and I cannot upload.
![wechatimg19](https://user-images.githubusercontent.com/32039206/31634441-8931225a-b291-11e7-8c41-c2ec8dbab8a8.jpeg)

**Comment by DenisCarriere** [comment]
Maybe you've been blocked? It says to read the message in OpenStreetMap website, what does it say?

**Comment by Carap1209** [comment]
Yes, it was. But now my account got unlocked. Thanks for your suggestion.
